transcom/mymove
# B 21584 fix email domain int

## [B-21584](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A1025327)

## Summary

Fixes email domain issue of multiple '.' not being allowed.

## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

1. Go to Office login page
2. Click "Request Account"
3. Fill in all information
4. Confirm that emails like "test@us.navy.mil" are valid including any other emails that end in .edu, .gov and .mil

### Frontend


## Screenshots
Invalid Email
<img width="566" alt="Screenshot 2024-10-24 at 3 39 00 PM" src="https://github.com/user-attachments/assets/dea5d70b-a25e-401a-a650-d5b2d4e74108">

Valid Emails
<img width="561" alt="Screenshot 2024-10-24 at 3 38 54 PM" src="https://github.com/user-attachments/assets/6c1218f6-0782-4723-91f0-0260dd2133b2">
<img width="478" alt="Screenshot 2024-10-24 at 3 39 10 PM" src="https://github.com/user-attachments/assets/e2dc5248-35bf-4f78-91eb-d038061a48ac">
